app/views/shared/_header.html.erb

```diff
@@ -1,63 +1,72 @@
 <header class="navbar navbar-expand navbar-light bg-light flex-row">
-  <%= link_to root_path, class: 'navbar-brand' do %>
-    <%= image_tag "feijoa_icon.png", class: 'rounded', height: 40, width: 40, alt: 'Feijoa home' %>
-    Feijoa
-  <% end %>
-  <ul class="navbar-nav">
-    <% if current_admin_user.present? %>
-      <li class="nav-item">
-        <%= link_to 'Home', authenticated_admin_user_root_path %>
-      </li>
-      <li class="nav-item">
-        <%= link_to 'Data', admin_root_path %>
-      </li>
+  <div class="container">
```

Comment:
This from a comment by Ross yesterday - keeps the navbar contents from filling the full width